gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 22 15:05:06 MDT 2015
+#Mon Apr 04 14:20:24 MDT 2016
```

Comment:
Haha, in response to this, and the above comment, I just have a wrapper sitting around I copy into a project.

I will update my wrapper for future projects, but I don't think having a slightly older version is a big deal.  Only reason I upgraded this is because I hate having to always re-download the wrapper when ever I go to a new project, lol.